### PR TITLE
Education Card & UI Updates

### PR DESCRIPTION
**Pull request by AtelyPham**
## Summary of changes
_Provide a detailed description of proposed changes._
- Update the hero heading font size on the marketing page
- Add transition when closing the banner
- Update bridge background images
- Add a new Education Card
- Update empty note account tables to match new designs
- Remove Arbitrum network because of deprecation

### Proposed area of change
_Put an `x` in the boxes that apply._

- [x] `apps/bridge-dapp`
- [ ] `apps/stats-dapp`
- [x] `apps/webbsite`
- [x] `libs/webb-ui-components`

### Reference issue to close (if applicable)
_Specify any issues that can be closed from these changes (e.g. Closes #233)._
- Closes https://github.com/webb-tools/webb-dapp/issues/979
- Closes https://github.com/webb-tools/webb-dapp/issues/1004
- Closes https://github.com/webb-tools/webb-dapp/issues/1001

### Screen Recording
_If possible provide a screen recording of proposed change._

https://user-images.githubusercontent.com/60747384/219437057-0c7a4cbe-d421-426b-8882-6a74c3f8ea23.mp4

### CODE REVIEW

File: apps/bridge-dapp/src/components/EducationCard/EducationCard.tsx [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+      description: 'Make a deposit and choose a destination chain',
+    },
+    {
+      title: 'Switch Networks',
+      description: 'Hop over to the destination chain',
+    },
+    {
+      title: 'Withdraw',
+      description: 'Withdraw using fixed amounts for max privacy',
+    },
```
Comment (dutterbutter) [inline apps/bridge-dapp/src/components/EducationCard/EducationCard.tsx:42]:
```suggestion
    {
      title: 'Create Note Account',
      description: 'Create a Note account to start transacting using your connected MetaMask address.',
    },
    {
      title: 'Select Token and Amount',
      description: 'Choose the token you wish to deposit and specify the deposit amount.',
    },
    {
      title: 'Select Desired Chains',
      description: 'Select the source and destination chains for your deposit.',
    },
    {
      title: 'Confirm Deposit',
      description: 'Review and confirm the details of your deposit before submitting it.,
    },
```

**Comment by devpavan04** [comment]
bg inconsistency between dark/light mode toggle. I think light mode bg placement looks better where asset icons are away from the text. Any thoughts?

https://user-images.githubusercontent.com/53374218/219441311-4f4d588c-9f96-4b61-9be6-ae20d1ef794a.mov

File: apps/bridge-dapp/src/components/EducationCard/EducationCard.tsx [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
```
Comment (dutterbutter) [inline apps/bridge-dapp/src/components/EducationCard/EducationCard.tsx:70]:
```suggestion
    {
      title: 'Connect to Destination chain',
      description: 'To withdraw your tokens, switch to the destination chain where you have a balance of the token you want to withdraw.',
    },
    {
      title: 'Specify the Withdrawal Amount',
      description: 'Choose the amount you want to withdraw. For maximum privacy, we recommend using fixed withdrawal amounts.',
    },
    {
      title: 'Choose a Relayer',
      description: 'To maximize your privacy, select a relayer to facilitate your transaction.',
    },
    {
      title: 'Confirm Withdraw',
      description: 'Review and confirm the details of your withdraw before submitting it.',
    },
```

**Comment by AtelyPham** [comment]
Good catch here 💯 @devpavan04. I'll reach out to the design team to get consistent backgrounds between light/dark modes.

**Review by dutterbutter — CHANGES_REQUESTED** [review]
Some copy suggestions for deposit / transfer / withdraw.

**Comment by dutterbutter** [comment]
@AtelyPham in your next commit can you please remove this line: https://github.com/webb-tools/webb-dapp/blob/df46e931256bde898e8164461b5c0af7ab31305f/.github/workflows/deploy-webbsite-dev.yml#L97 

Its causing an error with GA for webbsite deployment link.

**Comment by AtelyPham** [comment]
Would you please help me to check why the CI failing here, @dutterbutter?

File: apps/bridge-dapp/src/components/EducationCard/EducationCard.tsx [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+    href: 'https://docs.webb.tools/docs/dapps/hubble-bridge/overview/',
+    Icon: UsageGuideIcon,
+  },
+  {
+    label: 'Get Started',
+    getLink: (tab: EducationCardProps['currentTab']) => {
+      return `https://docs.webb.tools/docs/dapps/hubble-bridge/usage-guide/${tab.toLowerCase()}/`;
+    },
+    Icon: UsageGuideIcon,
+  },
```
Comment (dutterbutter) [inline apps/bridge-dapp/src/components/EducationCard/EducationCard.tsx:21]: These should be switched. Usage guides should be pointed to the dedicated operation (e.g. Deposit, Withdraw, Transfer), and Getting Started pointed to the Overview docs page.

**Review by dutterbutter — CHANGES_REQUESTED** [review]
Just one other change otherwise LGTM

**Comment by monaiuu** [comment]
When the browser go into smaller sizes (i.e., split screen), the education component overlaps with the bridge control card. To address this issue, lets make sure the education component is set up in flex and responsive to smaller screen sizes. 

Screenshot 2023-02-18 at 3 50 05 AM

**Comment by dutterbutter** [comment]
@AtelyPham I resolved the issue with the links and updated to transfer copy without a relayer.

**Comment by AtelyPham** [comment]
Fixed overlap issue on education card by flexing the width. @monaiuu Please review again whenever you have a chance, thank you 🙏.

https://user-images.githubusercontent.com/60747384/219790452-64616fd6-9de5-49c3-bd86-6049b059d874.mp4

**Comment by monaiuu** [comment]
> Fixed overlap issue on education card by flexing the width. @monaiuu Please review again whenever you have a chance, thank you 🙏.

Looks good!! Thank you for the quick fix!

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**